package.json

```diff
@@ -3,18 +3,20 @@
   "version": "1.3.1",
   "types": "types/index.d.ts",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
   "files": [
     "src",
     "dist",
-    "types/index.d.ts"
+    "types"
   ],
   "description": "A React library for hCaptcha",
   "scripts": {
     "start": "webpack-dev-server --config ./webpack.config.js --mode development",
     "test": "jest",
     "watch": "babel src -d dist --copy-files --watch",
     "transpile": "babel src -d dist --copy-files",
-    "build": "npm run transpile",
+    "build": "rm -rf dist && npm run transpile && npm run build:esm",
+    "build:esm": "cross-env BABEL_ENV=esm babel src -d dist/esm --copy-files",
```

Comment:
Nice cross env script. Shall the above `rm -rf` also be using a cross os command?